espnet2/tasks/abs_task.py

```diff
@@ -562,7 +562,7 @@ class ArgumentDefaultsRawTextHelpFormatter(
         group.add_argument(
             "--use_wandb",
             type=str2bool,
-            default=False,
+            default=True,
```

Comment:
Sorry, I didn't realize the default is True.
I'll change this to False because wandb causes an exception when no http connections exist.

Reply:
Sorry, my bad. It seems to work, so there are no problems.

Reply:
Hmm, sorry, after all, it doesn't seem to work when there are  http connection error, but the case must work for use case.
Is it correct? If so, I'll change the default to False.

Reply:
Yes wandb will print http error if it is not able to connect but that will not stop the training. If you decide to set `use_wandb = False` by default, please include this line in the readme so that users know how to enable it.
```
Once installed, run `wandb login` to enable tracking runs using W&B. To enable W&B tracking, set `use_wandb` to True
```

Reply:
In my environment, it tries to connect, probably until some timeout? and it doesn't raise error.  I'll change the default to False.